<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralwidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>781</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelChoose">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CHOOSE:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelHumanChooses">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>HUMAN CHOOSES:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="humanChoice">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>150</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelComputerIS">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>COMPUTER IS:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="choice">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>103</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>RANDOM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SMART</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelRounds">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ROUNDS:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="rounds">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>103</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="startGameButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>START NEW GAME</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelComputerChoice">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>280</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>COMPUTERS CHOICE:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="computerChoice">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>270</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="theWinner">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>330</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;THE WINNER:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="winner">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>320</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatistics">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>360</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;STATISTICS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelComputerWins">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>400</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>COMPUTER WINS:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelHumanWins">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>450</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>HUMAN WINS:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTies">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>490</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TIES:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="computerWins">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>390</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="humanWins">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>440</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ties">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>490</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rock">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ROCK</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="paper">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>120</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PAPER</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="scissors">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>120</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>SCISSORS</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRound">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;ROUND:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtRoundNumber">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="nextRound">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>330</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>START NEXT ROUND</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelHuman">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;HUMAN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelComputeris">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>201</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;COMPUTER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="predHuman">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>220</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelPerdictedHumanChoice">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PREDICTED HUMAN CHOICE:</string>
     </property>
    </widget>
   </widget>
   <widget class="QMenuBar" name="menubar_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>800</width>
      <height>22</height>
     </rect>
    </property>
    <widget class="QMenu" name="menuRockPaperScissors">
     <property name="title">
      <string>RockPaperScissors</string>
     </property>
    </widget>
    <addaction name="menuRockPaperScissors"/>
   </widget>
   <widget class="QStatusBar" name="statusbar_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>3</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
